src/components/WorkOrder/Appointment/AppointmentView.js

```diff
@@ -63,6 +73,34 @@ const AppointmentView = ({ workOrderReference }) => {
     setLoading(false)
   }
 
+  const makePostRequest = async (
+    formData,
+    comments,
+    dateSelected,
+    slot,
+    commentsForJobStatus
+  ) => {
+    setComments(comments)
+    setDateSelected(dateSelected)
+    setSlot(slot)
+
+    setLoading(true)
+
+    try {
+      await postScheduleAppointment(formData).then(() =>
+        postJobStatusUpdate(commentsForJobStatus)
+      )
+    } catch (e) {
+      console.log(e)
+      setScheduleAppointmentSuccess(true)
```

Comment:
Is the correct setting? I'd have thought an error would result in this being set to `false`..

Reply:
oh...I forgot to mention, I put it there since we can't test successful request , so I can see how scheduleAppointmentSuccess looks like 
will remove it from there later